### ISSUE TITLE
Unable to add, edit or move tiles with keyboard only

### ISSUE THREAD

**Issue by briandhs** [Adam-it/TilesLinksForSPOnline#24]
**Describe the bug**
Unable to add, edit or move tiles with keyboard only.

**To Reproduce**
1. Navigate to ‘Add new link’ via keyboard.

2. Press enter.  Panel will open

3. Press tab, focus will move to tile (if there is one) instead of the form field in the panel

 

1. Navigate to tile via keyboard.  Edit and move buttons do not show and are obviously not navigable.

**Expected behavior**
Keyboard users should be able to at least add and edit tiles.

**Comment by Adam-it** [comment]
thanks @briandhs for the comment 👍
TBH the main cool feature of this webpart is actually the drag and drop functionality which is impossible to do with keyboard anyway so I never developed this solution with possibility to use it only using keyboard 😉.
But I do agree it would be a nice enhancement to allow to generally use some features of the webpart with keyboard only 🤩
As I do not consider this as bug but just something which was never developed and in initial idea of this webpart I will add it to my to do list to develop in future (hard to say if I will find time to do it quite soon as currently before Christmas I am quite full with work)
Also since this is an open source project maybe you would like to help me out 👍 and try to develop it on your own and open a PR to improve this webpart?
